### PR TITLE
HBW-98 Add disable_if condition

### PR DESCRIPTION
**Pull request by sugarfree1**

### CODE REVIEW

File: hbw/app/javascript/packs/hbw/components/form/string.js.jsx [hunk 1]
```diff
@@ -61,7 +61,8 @@ modulejs.define('HBWFormString', ['React'], (React) => {
           <input {...opts}
                  ref={(i) => { this.input = i; }}
                  className={`form-control ${!this.state.valid && ' invalid'}`}
-                 value={this.state.visualValue} />
+                 value={this.state.visualValue}
+                 disabled={this.props.disabled} />
```
Comment (TimAle) [inline hbw/app/javascript/packs/hbw/components/form/string.js.jsx]: Don't we want to make it `readOnly` to allow input content selection with cursor?

**Comment by sugarfree1** [comment]
Updated

File: hbw/app/javascript/packs/hbw/components/form/string.js.jsx (hunk 1 above)
Comment (sugarfree1) [reply hbw/app/javascript/packs/hbw/components/form/string.js.jsx]: Good point

**Comment by sugarfree1** [comment]
Updated

**Comment by sugarfree1** [comment]
Updated

**Pull request merged**